projects/components/src/beta-tag/beta-tag.module.ts

```diff
@@ -1,8 +1,11 @@
+import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
+import { LabelTagModule } from '../label-tag/label-tag.module';
 import { BetaTagComponent } from './beta-tag.component';
 
 @NgModule({
   declarations: [BetaTagComponent],
+  imports: [CommonModule, LabelTagModule],
```

Comment:
Don't see anything used from common module here

Reply:
CommonModule is for ngStyle

Reply:
ngStyle is used by label-tag rather than beta directly so is a concern of `LabelTagModule`

Reply:
oh. I looked at the wrong place. Sorry. Yes. Its not required here. Will rmeove.